pkgs/development/libraries/gnutls/default.nix

```diff
@@ -49,8 +49,10 @@ stdenv.mkDerivation {
     sed '2iexit 77' -i tests/{pkgconfig,fastopen}.sh
     sed '/^void doit(void)/,/^{/ s/{/{ exit(77);/' -i tests/{trust-store,psk-file}.c
     sed 's:/usr/lib64/pkcs11/ /usr/lib/pkcs11/ /usr/lib/x86_64-linux-gnu/pkcs11/:`pkg-config --variable=p11_module_path p11-kit-1`:' -i tests/p11-kit-trust.sh
-  '' + lib.optionalString stdenv.hostPlatform.isMusl '' # See https://gitlab.com/gnutls/gnutls/-/issues/945
+  '' + lib.optionalString (stdenv.hostPlatform.isMusl && lib.versionOlder version "3.7") '' # See https://gitlab.com/gnutls/gnutls/-/issues/945
     sed '2iecho "certtool tests skipped in musl build"\nexit 0' -i tests/cert-tests/certtool
+  '' + lib.optionalString (stdenv.hostPlatform.isMusl && lib.versionAtLeast version "3.7") ''
```

Comment:
I think all these version switches are here historically. We only have a single version of it and they can be removed.
At minimum we should introduce new ones.

Reply:
fixed in 64e1ca3e1adb23e09386cdd7a85047f525f20d86